TOOLS/ce/src/main/scala/org.opalj.ce/ConfigNode.scala

```diff
@@ -0,0 +1,16 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package ce
+
+/**
+ * Trait for representing the config structure
+ * Provides a Hull that the inheriting classes have to implement to function
```

Comment:
Apart from the typo ("hull"), I don't even get what this is meant to say. Isn't this just what a `trait` is anyway?

Reply:
It is.
I removed it.